internal/format/asciidoc_document.go

```diff
@@ -149,13 +149,33 @@ const (
 	{{ end -}}
 	`
 
+	asciidocDocumentModulecallsTpl = `
+	{{- if .Settings.ShowModulecalls -}}
+		{{ indent 0 "=" }} Modulecalls
+		{{ if not .Module.Modulecalls }}
+			No Modulecalls.
+		{{ else }}
+			the following Modules are called:
+			{{- range .Module.Modulecalls }}
+
+				{{ indent 1 "=" }} {{ name .Name }}
+
+				Source: {{ .Source }}
+
+				Version: {{ .Version }}
+			{{- end }}
+		{{ end }}
+	{{ end -}}
+	`
+
 	asciidocDocumentTpl = `
 	{{- template "header" . -}}
 	{{- template "requirements" . -}}
 	{{- template "providers" . -}}
 	{{- template "resources" . -}}
 	{{- template "inputs" . -}}
 	{{- template "outputs" . -}}
+	{{- template "modulecalls" . -}}
```

Comment:
Please move modules section right after providers section. This applies to asciidoc, markdown, and pretty.